src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -21,9 +21,13 @@ We use Docker to replicate the production environment for the local deployment o
 
 Use the `appwrite run functions` command to develop your function locally.
 
+| parameter       | Description                                                                                                |
```

Comment:
Also `--no-reload`. When set, functions will not hot reload. Simply build, start, and keep running. Any changes to your code wont cause restart or anything.